internal/experiment/webconnectivityqa/dnshijacking.go

```diff
@@ -33,9 +33,9 @@ func dnsHijackingToProxyWithHTTPURL() *TestCase {
 			StatusCodeMatch: true,
 			HeadersMatch:    true,
 			TitleMatch:      true,
-			XStatus:         2, // StatusSuccessCleartext
-			XDNSFlags:       0,
-			XBlockingFlags:  32, // AnalysisBlockingFlagSuccess
+			XStatus:         2,  // StatusSuccessCleartext
+			XDNSFlags:       4,  // AnalysisDNSFlagUnexpectedAddrs
+			XBlockingFlags:  33, // AnalysisBlockingFlagDNSBlocking | AnalysisBlockingFlagSuccess
```

Comment:
Even when we're successful, we want to note that we're not sure about the addresses.